modules/central-configuration-options-operator.adoc

```diff
@@ -96,7 +96,7 @@ a| Setting this parameter will not deploy Central DB, and Central will connect u
 This parameter has the following constraints:
 
 * Connection string must be in keyword/value format as described in the PostgreSQL documentation. For more information, see the links in the *Additional resources* section.
-* Only PostgreSQL 13 is supported.
+* Only PostgreSQL 15 is supported.
```

Comment:
This is BYODB, so I don't think this is true.  Maybe something like:

Postgres 13 support is deprecated and will be removed.  Postgres 15 is the supported and preferred version.